packages/frontend/package.json

```diff
@@ -18,9 +18,14 @@
   },
   "scripts": {
     "start": "npm run clean && npm run build:watch",
-    "build": "cross-env NODE_ENV=development webpack",
-    "build:prod": "cross-env NODE_ENV=production webpack",
-    "build:watch": "cross-env NODE_ENV=development webpack --watch",
+    "build": "ng run frontend:build-elements:production",
+    "build:watch": "cross-env NODE_ENV=development npm run build -- --watch",
```

Comment:
I don't think we need cross-env any longer no this is building with the CLI. The `cross-env NODE_ENV=development` could be remove throughout.